src/models/zone.ts

```diff
@@ -1,9 +1,9 @@
+import * as PIXI from "pixi.js";
 import { ICollider } from "./collider";
-import { User } from "./user";
 
 export interface IZone extends ICollider {
-  tryInteract: (user: User) => void;
-  tryPlace: (user: User, spotID: number) => void;
+  tryInteract: (user: PIXI.Sprite) => void;
```

Comment:
Avoiding circular dependency (and I think this is reasonable because we could in theory add other interactable sprites, not just users)